src/routes/album.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+const {createAlbum, getAlbums, getAlbumById, updateAlbumById, patchAlbumById, deleteAlbumById} = require('../controllers/album')
+const albumRouter = express.Router()
+
+
+albumRouter.post('/artists/:id/albums', createAlbum)
+
+albumRouter.get('/albums', getAlbums)
+
+albumRouter.get('/albums/:id', getAlbumById)
+
+albumRouter.put('/albums/:id', updateAlbumById)
+
+albumRouter.patch('/albums/:id', patchAlbumById)
+
+albumRouter.delete('/albums/:id', deleteAlbumById)
```

Comment:
Following on from the comments in app.js the base route of `/albums` should be added there. Here you only need anything that comes after `/albums`
```
albumRouter.get('/', getAlbums)

albumRouter.get('/:id', getAlbumById)

albumRouter.put('/:id', updateAlbumById)

albumRouter.patch('/:id', patchAlbumById)

albumRouter.delete('/:id', deleteAlbumById)
```